.github/workflows/nodejs-ci.yml

```diff
@@ -216,11 +236,29 @@ jobs:
         if: ${{ !cancelled() && !contains( inputs.excluded_jobs, 'build' )  }}
         run: |
           ${{ inputs.build_command }}
+      - name: Developer Tests
+        id: developer_tests
+        if: ${{ always() && inputs.run_dev_test && contains(fromJson(inputs.developer_test_branch), github.event.pull_request.base.ref) }}
```

Comment:
Try withou using fromJson, it should work. fromJson is not needed.

Reply:
done